staging/publishing/rules.yaml

```diff
@@ -629,9 +629,22 @@ rules:
       branch: master
       dir: staging/src/k8s.io/kubectl
     name: master
+    dependencies:
+    - repository: api
+      branch: master
+    - repository: apimachinery
+      branch: master
+    - repository: client-go
+      branch: master
   - source:
       branch: release-1.15
       dir: staging/src/k8s.io/kubectl
     name: release-1.15
     go: 1.12.5
-
+    dependencies:
+    - repository: api
+      branch: release-1.15
+    - repository: apimachinery
+      branch: release-1.15
+    - repository: client-go
```

Comment:
From go.mod, it looks like client-go is not a dependency and can be removed?

Reply:
loooking, thanks.  there is a slew of these PRs atm (moving pkg/kubectl/util/* to staging), i _think_ we all are introducing this same change.. so whoever's merges first has to be correct lol

Reply:
looks like it, yes.  @damemi  @pswica @seans3

Reply:
removed